src/test/resources/no/nav/tilleggsstonader/sak/vilkår/slå_sammen_periode_grunnlag_ytelse.feature

```diff
@@ -0,0 +1,78 @@
+# language: no
+# encoding: UTF-8
+
+Egenskap: Slå sammen periode grunnlag ytelse
```

Comment:
Savner tester med litt mer vanlige caser: 

F.eks. 
- "Slå sammen påfølgende perioder av alle typer" hvor du kan ha to av hver type som forventes at slås sammen
- "Slå sammen overlappende perioder av alle typer" hvor du også har hver type og at de slås sammen.

Mulig det er litt overkill med alle typene på begge, men viktig å teste at grunnleggende funksjonalitet er på plass, slik at man ved endringer senere oppdager at funksjonalitet ikke er som forventet lenger. Også veldig viktig å teste de rare casene og så 👏

Reply:
Jeps, jeg fikser 👍